### ISSUE TITLE
Optimize themedd_page_header for $title usage

### ISSUE THREAD

**Issue by flowdee** [easydigitaldownloads/themedd#191]
https://github.com/easydigitaldownloads/themedd/blob/2403b8844197cbef9a9b72738fe5ad842d450245/includes/template-tags.php#L255

Not sure if I'm missing something, but it seems like $title is never been used?

**Comment by amdrew** [comment]
A custom title can be passed in via the `$args` array, otherwise it will use the page title via `get_the_title()`.

**Comment by flowdee** [comment]
Yeah sure, but the var $title is never been used in your code. To me, it seems like a piece of duplicate code, or am I wrong?

<img width="1100" alt="bildschirmfoto 2018-03-12 um 07 53 41" src="https://user-images.githubusercontent.com/2278756/37269420-81c1a27c-25ca-11e8-9b2f-eccfb75eab50.png">

**Comment by amdrew** [comment]
You're right, that doesn't look correct at all

**Comment by flowdee** [comment]
Additionally wouldn't it make sense to add a few more standard cases here, such as:

- `is_category()` > `single_cat_title()`
- `is_tag()` > `single_tag_title()`
- `is_tax()` > `single_term_title( '', false)`
- `is_post_type_archive()` > `post_type_archive_title('', false)`
- `is_search()` > `sprintf( esc_html__( 'Search Results for %s', 'themedd' ), get_search_query() )`
